test/tap/tests/test_admin_stats-t.cpp

```diff
@@ -0,0 +1,180 @@
+/**
+ * @file test_admin_stats-t.cpp
+ * @brief This tests the Statistics module and its lookup code and tables.
```

Comment:
A @details section detailing which each of the tests performed is aim to verify would be nice, this way there is no need to reading the test code for knowing everything it aims to check.

Reply:
OK. I was just going off of example in test_admin_prometheus_metrics_dump-t... as a starting point. I see now looking at another test what you mean. I'll add that in there.